bigquery.js

```diff
@@ -0,0 +1,93 @@
+var BigQueryClient = require('google-cloud/node_modules/@google-cloud/bigquery');
+
+const projectData = require('./auth.json');
+
+const config = {
+  projectId: projectData['project_id'],
+  keyFilename: './auth.json'
+};
+
+const ds = 'atomic';
+
+const PARTITION_FIELDS = ['updated_at', 'created_at', 'fired_at', 'ts'];
+
+var BigQuery = function() {
+  this.conn = BigQueryClient(config);
+};
+
+BigQuery.prototype.insertInto = function(tableName, rows, options) {
+  //Here I should get the table based on name
+  // and insert in the right partition whenever the table is partitioned
+  rows.forEach((row) => {
+    // First, we get the table object with all available information
+    this.table(tableName, function(err, table) {
```

Comment:
Consider to use arrow functions (`this.table(tableName, => (err, table) { ...`) instead of classi functions, because you dont need the inner function context 👍

Reply:
Did you mean `this.table(tableName, (err, table) => { ...`?

Reply:
I didn't get the reason why using arrow functions do remove the need of the inner function context, @renatomenegasso; do you care to explain it a bit more, please?

Reply:
For me, has two main points:
- Arrow functions are a little bit lighter than normal functions, because they dont have the `prototype` property (you cannot use `new` to instantiate an arrow function), and don't have `arguments` special variable. If you don't need them, it'll a bit lighter
- Less noise in the code: you can write a shorter code 👍